packages/components/src/input-control/backdrop.tsx

```diff
@@ -7,12 +7,17 @@ import { memo } from '@wordpress/element';
  */
 import { BackdropUI } from './styles/input-control-styles';
 
-function Backdrop( { disabled = false, isFocused = false } ) {
+function Backdrop( {
+	disabled = false,
+	isBorderless = false,
+	isFocused = false,
+} ) {
 	return (
 		<BackdropUI
 			aria-hidden="true"
 			className="components-input-control__backdrop"
```

Comment:
Technically, we can already just use this `.components-input-control__backdrop` selector to override border styles from any consumer. Still, I think it's better to control this centrally because it's intertwined with the border styles in  the focused and disabled states, as you can see in the `input-control-styles.tsx` file.